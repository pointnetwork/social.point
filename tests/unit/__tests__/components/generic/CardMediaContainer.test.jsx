import React from "react";
import renderer from 'react-test-renderer';

import CardMediaContainer from '../../../../../src/components/generic/CardMediaContainer';

import image from "../../assets/header-pic.jpg"

describe("CardMediaContainer", () => {

    it('should render', () => {
        const component = renderer.create(
            <CardMediaContainer media={image}></CardMediaContainer>
        )
        expect(component.toJSON()).toMatchSnapshot();
    });

});
