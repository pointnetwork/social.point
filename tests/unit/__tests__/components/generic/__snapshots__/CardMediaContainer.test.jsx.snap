// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CardMediaContainer should render 1`] = `
<div
  className="MuiBox-root MuiBox-root-8 makeStyles-root-1"
>
  <div
    className="makeStyles-frame-2"
  >
    <div
      className="MuiBox-root MuiBox-root-9"
    >
      <div
        className="MuiCircularProgress-root MuiCircularProgress-indeterminate"
        role="progressbar"
        style={
          Object {
            "height": 40,
            "width": 40,
          }
        }
      >
        <svg
          className="MuiCircularProgress-svg"
          viewBox="22 22 44 44"
        >
          <circle
            className="MuiCircularProgress-circle MuiCircularProgress-circleIndeterminate"
            cx={44}
            cy={44}
            fill="none"
            r={20.2}
            strokeWidth={3.6}
            style={Object {}}
          />
        </svg>
      </div>
      <div
        className="MuiBox-root MuiBox-root-10"
      >
        <svg
          aria-hidden={true}
          className="MuiSvgIcon-root"
          focusable="false"
          viewBox="0 0 24 24"
        >
          <path
            d="M19 5v14H5V5h14m0-2H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2zm-4.86 8.86l-3 3.87L9 13.14 6 17h12l-3.86-5.14z"
          />
        </svg>
      </div>
    </div>
  </div>
</div>
`;
